### DISCUSSION TITLE
How I combine multiple apps in one?

### DISCUSSION THREAD

**Discussion by bylly1** [vercel/turborepo#3470] [Help]
I have multiple vite react apps and I want to combine in a single app: 
home: http://localhost:3000
app1: http://localhost:3001
app2: http://localhost:3002

For routing inside the apps I'm using react-router v6.4

I want the following thing: 
`http://localhost` to be redirected to `home` app
`http://localhost/app1 to be redirected to `http://localhost:3001`  (app1)
`http://localhost/app2` to be redirected to `http://localhost:3002` (app2)

For that I configured a nginx reverse proxy like that but didn t work

```
server {
 location \ {
   proxy_pass http://localhost:3000;
 }
location \app1 {
  proxy_pass http://localhost:3001;
}
location \app2 {
  proxy_pass http://localhost:3002;
}
}
```
All the request goes to the `location \` . I tried also to pass the location like this `location = \ {...}` but still don't work. 

Is somebody here that faced the problem?

**Comment by sheik-mostafizur** [comment]
Same problem, Did you find any solution?

https://github.com/edust-org/edust/issues/64

**Comment by mariosffx** [comment]
That's the Module Federation Conept:
https://webpack.js.org/concepts/module-federation/

You can find an example here:
https://github.com/module-federation/module-federation-examples
